src/main/java/edu/cornell/kfs/sys/service/BatchFileDirectoryService.java

```diff
@@ -4,12 +4,14 @@
 import org.kuali.rice.core.api.util.KeyValue;
 import org.springframework.cache.annotation.Cacheable;
 
-import java.io.File;
 import java.util.List;
 
 public interface BatchFileDirectoryService {
 
-    @Cacheable(value= SystemOptions.CACHE_NAME, key="'buildDirectoryKeyValuesList' + #p0")
-    List<KeyValue> buildDirectoryKeyValuesList(List<File> rootDirectories);
+    @Cacheable(value= SystemOptions.CACHE_NAME, key="'buildStagingDirectories'")
+    List<KeyValue> buildBatchFileStagingDirectories();
+
+    @Cacheable(value= SystemOptions.CACHE_NAME, key="'buildStagingAndReportsDirectories'")
+    List<KeyValue> buildBatchFileLookupDirectories();
```

Comment:
The cache key is using a value other than the name of the method, unlike the method above it. Is this intentional?